packages/contracts-bedrock/contracts/L2/GasPriceOracle.sol

```diff
@@ -56,21 +62,6 @@ contract GasPriceOracle is Semver {
      */
     uint256 public constant decimals = 6;
 
-    /**
-     * @notice Emitted when the overhead value is updated.
-     */
-    event OverheadUpdated(uint256 overhead);
-
-    /**
-     * @notice Emitted when the scalar value is updated.
-     */
-    event ScalarUpdated(uint256 scalar);
```

Comment:
Hmmm this is a weird case. It changes the ABI, meaning that if you want to scan for the legacy events then you need to use a legacy package that exports the correct ABI. I'm not super opinionated on this though. I lean on the side of deleting this code since it isn't being used anymore.

Reply:
Acknowledging your thoughts, going to resolve this comment thread so this PR auto merges